### ISSUE TITLE
Flip.js Latest version issue

### ISSUE THREAD

**Issue by 13papatya** [codrops/OneElementScroll#2]
There's an issue with the header full-screen image with the latest version of Flip, it seems this one is using 3.11.4 while the latest version is 3.12.5
Is it possible to provide an update for the issue?

**Comment by crnacura** [comment]
Yes, the team is aware of this bug and are on it! I'll push an update then, thanks for opening this!
